src/components/BlogsCarouselHome/index.jsx

```diff
@@ -8,24 +8,40 @@ import "./index.scss";
 import "slick-carousel/slick/slick.css";
 import "slick-carousel/slick/slick-theme.css";
 import SliderArrow from "../SliderArrow";
+import i18next from "i18next";
 
-const POSTS_PER_PAGE = 9;
+const POSTS_PER_PAGE = 18;
```

Comment:
actually...

https://developer.wordpress.org/rest-api/reference/posts/#list-posts

you guys should be using the "tags" param in the API. and when the language switches, you should be calling fetch() again.

its wrong for this to say 18 POSTS_PER_PAGE. are there 18 posts per page? nope

Reply:
i don't see it locally :/

Reply:
removed